### PR TITLE
Add latest package

### PR DESCRIPTION
**Pull request by acidsennui**
pt: https://www.pivotaltracker.com/n/projects/2152594/stories/159699819
contain: https://github.com/aeternity/epoch/pull/1435

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -133,12 +133,18 @@ references:
   upload_build_artifacts: &upload_build_artifacts
     run:
       name: Upload build artifacts to central repository (AWS)
-      command: aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+      command: |
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
```
Comment (dincho) [inline .circleci/config.yml]: Let's have latest build as well, we'll probably use it on integration env setups

File: .circleci/config.yml [hunk 2]
```diff
@@ -133,12 +133,20 @@ references:
...
+        aws s3api put-object-acl --acl public-read --bucket ${S3_BUILDS_BUCKET:?} --key `basename ${PACKAGE_TARBALL:?}
+        aws s3api put-object-acl --acl public-read --bucket ${S3_BUILDS_BUCKET:?} --key epoch-latest-ubuntu-x86_64.tar.gz
 
   upload_release_artifacts: &upload_release_artifacts
     run:
       name: Upload release artifacts to central repository (AWS)
-      command: aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/
+      command: |
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/epoch-latest-ubuntu-x86_64.tar.gz
```
Comment (dincho) [inline .circleci/config.yml]: Hardcoding the arch is not very future proof, what if we decide to publish the OSX packages (with the same CircleCI task) ?

File: .circleci/set_package_path.sh [hunk 3]
```diff
@@ -14,4 +14,5 @@ if [[ -n $CIRCLE_TAG && $CIRCLE_TAG =~ ^v([0-9]+\.[0-9]+\.[0-9]+)$ ]]; then
 fi
 
 PACKAGE_TARBALL=${PACKAGES_DIR:?}/epoch-${VERSION}-${PKG_SUFFIX}.tar.gz
+exho "export PKG_SUFFIX=${PKG_SUFFIX}" >> $BASH_ENV
```
Comment (lucafavatella) [inline .circleci/set_package_path.sh]: exho?

File: .circleci/config.yml [hunk 4]
```diff
@@ -133,12 +133,20 @@ references:
   upload_build_artifacts: &upload_build_artifacts
     run:
       name: Upload build artifacts to central repository (AWS)
-      command: aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+      command: |
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/epoch-latest-ubuntu-x86_64.tar.gz
```
Comment (dincho) [inline .circleci/config.yml]: I would try `aws s3 cp s3://${S3_BUILDS_BUCKET:?}/${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/epoch-latest-${PKG_SUFFIX:?}.tar.gz`

File: .circleci/config.yml (hunk 2 above)
Comment (dincho) [reply .circleci/config.yml]: Could you please use `PKG_SUFFIX` all over the commands, and please also make it required by `${PKG_SUFFIX:?}`

**Pull request merged**